src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/helpers/SynchronisationHelpers.kt

```diff
@@ -0,0 +1,147 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.helpers
```

Comment:
Copied from TO Nomis

Reply:
Do you actually need these classes? I cant see where they are used.

Reply:
I needed SynchronisationHelpers - and then that needed RetryQueueService - I'll shrink down the SynchronisationHelpers for just what I need and remove the other file